renovate.json

```diff
@@ -1,17 +1,44 @@
 {
-  "baseBranches": ["next"],
```

Comment:
This will default to `develop` only. If we want `next` and `develop` you should update this to include both.

Reply:
I might need to throw another config in here to disable at base because I only want to enable the two groups below.